react/components/InputSearch/index.js

```diff
@@ -73,15 +84,19 @@ class InputSearch extends Component {
         onBlur={() => this.handleFocus(false)}
         onMouseEnter={() => this.handleHovering(true)}
         onMouseLeave={() => this.handleHovering(false)}
-        onKeyUp={e => e.key === 'Enter' && this.handleSubmit(e)}
+        onKeyUp={e => this.pressedEnter(e) && this.handleSubmit(e)}
         type="search"
         suffix={
           <div className="flex flex-row items-center">
             {this.props.value && (
               <span
-                tabIndex={0}
-                onClick={this.handleClickClear}
                 className="pointer mr4 c-muted-3"
+                onClick={this.handleClickClear}
+                onKeyPress={e =>
+                  this.pressedEnter(e) && this.handleClickClear(e)
+                }
```

Comment:
here you should use `this.handleKeyPress` instead of `this.pressedEnter` no?

you can turn it into a generic function that takes a function as a parameter and then you can pass `handleSubmit` or `handleClickClear`.

Reply:
great idea. done!